lib/agent.js

```diff
@@ -16,12 +17,9 @@ const kDestroyed = Symbol('destroyed')
 const kClosed = Symbol('closed')
 const kOnConnect = Symbol('onConnect')
 const kOnDisconnect = Symbol('onDisconnect')
-const kClients = Symbol('clients')
```

Comment:
Also, just drawing your attention to this as this was the only bit of actual undici code that I changed - this was so I could access the symbols I need to hook into the encapsulated agent `kClients` `Map`.